ConceptMap/ConceptMap-Genomics-specimen-final-sample.json

```diff
@@ -0,0 +1,136 @@
+  "targetUri": "http://snomed.info/sct",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "DNA",
+          "display": "DNA",
+          "target": [
+            {
+              "code": "258566005",
+              "display": "Deoxyribonucleic acid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "DNA/RNA",
+          "display": "DNA/RNA",
+          "target": [
+            {
+              "code": "258566005",
+              "display": "Deoxyribonucleic acid specimen (specimen)",
+              "equivalence": "narrower"
+            },
+            {
+              "code": "441673008",
+              "display": "Ribonucleic acid specimen (specimen)",
+              "equivalence": "narrower"
+            }
+          ]
+        },
+        {
+          "code": "RNA",
+          "display": "RNA",
+          "target": [
+            {
+              "code": "441673008",
+              "display": "Ribonucleic acid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "cDNA",
+          "display": "cDNA",
+          "target": [
+            {
+              "code": "70016006",
+              "display": "Complementary deoxyribonucleic acid (substance)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "ctDNA",
+          "display": "ctDNA",
+          "target": [
+            {
+              "equivalence": "unmatched"
+            }
+          ]
+        },
+        {
+          "code": "cfDNA",
+          "display": "cfDNA",
+          "target": [
+            {
+              "code": "70016006",
+              "display": "Cell free deoxyribonucleic acid (substance)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "cell-suspension",
+          "display": "Cell Suspension",
+          "target": [
+            {
+              "code": "702451000",
+              "display": "Cultured cells (substance)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "fixed-tissue-on-slide",
+          "display": "Fixed Tissue on Slide",
+          "target": [
+            {
+              "code": "787150001",
+              "display": "Stained slide of tissue section",
```

Comment:
This may need more thought as the source code does not necessarily mention whether the specimen is stained or unstained, maybe add both concepts, with equivalence narrower and add a comment to state that the appropriate code should be used depending on whether the slide is stained or unstained